python/doc/examples/reliability_sensitivity/reliability/plot_crossentropy.py

```diff
@@ -0,0 +1,328 @@
+print("Initial distribution:", distribution)
+
+# %%
+# Draw the limit state function :math:`g` to help to understand the shape of the limit state.
+
+# %%
+g = axialBeam.model
+graph = ot.Graph("Simple stress beam", "R", "F", True, "topright")
+drawfunction = g.draw([1.8e6, 600], [4e6, 950.0], [100] * 2)
+graph.add(drawfunction)
+view = viewer.View(graph)
+
+# %%
+# Create the output random vector :math:`Y = g(\vect{X})` with :math:`\vect{X} = [R,F]`.
+
+# %%
+X = ot.RandomVector(distribution)
+Y = ot.CompositeRandomVector(g, X)
+
+# %%
+# Create the event :math:`\{ Y = g(\vect{X}) \leq 0 \}`
+# -----------------------------------------------------
+
+# %%
+threshold = 0.0
+event = ot.ThresholdEvent(Y, ot.Less(), 0.0)
+
+# %%
+# Evaluate the probability with the Standard Space Cross Entropy
+# --------------------------------------------------------------
+
+# %%
+# We choose to set the intermediate quantile level to 0.35.
+
+# %%
+standardSpaceIS = otexp.StandardSpaceCrossEntropyImportanceSampling(event, 0.35)
+
+# %%
+# Now we can run the algorithm and get the results.
+
+# %%
+standardSpaceIS.run()
+standardSpaceISResult = standardSpaceIS.getResult()
+proba = standardSpaceISResult.getProbabilityEstimate()
+print("Proba Standard Space Cross Entropy IS = ", proba)
+print(
+    "Current coefficient of variation = ",
+    standardSpaceISResult.getCoefficientOfVariation(),
+)
+
+# %%
+# The length of the confidence interval of level :math:`95\%` is:
+
+# %%
+length95 = standardSpaceISResult.getConfidenceLength()
+print("Confidence length (0.95) = ", standardSpaceISResult.getConfidenceLength())
+
+# %%
+# which enables to build the confidence interval.
+
+# %%
+print(
+    "Confidence interval (0.95) = [",
+    proba - length95 / 2,
+    ", ",
+    proba + length95 / 2,
+    "]",
+)
+
+# %%
+# We can analyze the simulation budget.
+
+# %%
+print(
+    "Numerical budget: ",
+    standardSpaceISResult.getBlockSize() * standardSpaceISResult.getOuterSampling(),
+)
+
+# %%
+# We can also retrieve the optimal auxiliary distribution in Standard Space.
+
+# %%
+print(
+    "Auxiliary distribution in Standard space: ",
+    standardSpaceISResult.getAuxiliaryDistribution(),
+)
+
+
+# %%
+# Draw initial samples and final samples
+# --------------------------------------
+#
+
+# %%
+# First we get the auxiliary samples in Standard Space and we project them in Physical Space.
+
+# %%
+auxiliaryInputSamples = standardSpaceISResult.getAuxiliaryInputSample()
+auxiliaryInputSamplesPhysicalSpace = distribution.getInverseIsoProbabilisticTransformation()(
+    auxiliaryInputSamples
+)
+
+
+# %%
+graph = ot.Graph("Cloud of samples and failure domain", "R", "F", True, "topright")
+# Generation of samples with initial distribution
+initialSamples = ot.Cloud(
+    distribution.getSample(1000), "blue", "plus", "Initial samples"
+)
+auxiliarySamples = ot.Cloud(
+    auxiliaryInputSamplesPhysicalSpace, "orange", "fsquare", "Auxiliary samples"
+)
+# Plot failure domain
```

Comment:
When the [DrawEvent](https://github.com/mbaudin47/otbenchmark/blob/master/otbenchmark/DrawEvent.py) class will be available, this code will be much easier. In the meantime, the code you wrote is the simplest possible I guess.